src/Microsoft.ML.Core/Data/ServerChannel.cs

```diff
@@ -141,7 +141,7 @@ public void Acknowledge(Action<Bundle> toDo)
         /// Entry point factory for creating <see cref="IServer"/> instances.
         /// </summary>
         [TlcModule.ComponentKind("Server")]
-        public interface IServerFactory : IArgsComponent, IComponentFactory<IChannel, IServer>
```

Comment:
>IArgsComponent [](start = 42, length = 14)

feel free to remove this interface as well. It was temporary introduce for Neural nets entry points almost two years ago, and we no longer need it.

Reply:
I've removed `IArgsComponent` completely.